src/pipelines/epidemiology/fi_authority.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import datetime
+from typing import Dict, List
+from pandas import DataFrame
+from lib.pipeline import DataSource
+from lib.utils import table_rename
+
+
+_column_adapter = {
+    "date": "date",
+    "kunta": "subregion1_name",
+    "tapauksia": "total_confirmed",
+    "miehia": "total_confirmed_male",
+    "naisia": "total_confirmed_female",
+    "ika_0_9": "total_confirmed_age_00",
+    "ika_10_19": "total_confirmed_age_01",
+    "ika_20_29": "total_confirmed_age_02",
+    "ika_30_39": "total_confirmed_age_03",
+    "ika_40_49": "total_confirmed_age_04",
+    "ika_50_59": "total_confirmed_age_05",
+    "ika_60_69": "total_confirmed_age_06",
+    "ika_70_79": "total_confirmed_age_07",
+    "ika_80_": "total_confirmed_age_08",
+}
+
+
+# pylint: disable=missing-class-docstring,abstract-method
```

Comment:
I don't see a pylint configuration for this project. Is this for future use?

Reply:
I copied it from the link you sent me: https://github.com/open-covid-19/data/blob/master/src/.pylintrc

Although I need to tweak it a bit more, because `pylint` is now fighting with `black` over some indentation things.